heal-remedy.lic

```diff
@@ -0,0 +1,93 @@
+# quiet
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tendme
+=end
+
+custom_require.call(%w[common common-healing common-items])
+
+
+arg_definitions = [
+  [
+    { name: 'debug', regex: /debug/i, optional: true }
+  ]
+]
+
+args = parse_args(arg_definitions)
+$debug_mode_hr = args.debug
+
+$remedies = get_data('remedies')
+
+def wounds
+  wounds = DRCH.check_health
+  pause 1
+#  echo (wounds)
+  hurt = wounds.select{|level, _| level >= 4}.values.flatten
+  wound_grouping = { "right arm" => "limbs", "left arm" => "limbs", "left leg" => "limbs", "right leg" => "limbs", "right hand" => "limbs", "left hand" => "limbs", "right eye" => "eyes", "left eye" => "eyes" }
+  scar_grouping = { "head" => "face", "neck" => "face", "eyes" => "face", "chest" => "body", "abdomen" => "body", "back" => "body" }
+  hurt = hurt.map{ |raw| wound_grouping[raw] || raw}.uniq
+  scar = hurt.map{ |raw| scar_grouping[raw] || raw}.uniq
+  echo ("Scars: ") if $debug_mode_hr
+  if $debug_mode_hr
+    scar.each do |key| 
+      puts key
+    end
+  end
+  hurt.each do |key|
+    puts key if $debug_mode_hr
+    echo("In break loop") if $debug_mode_hr
+    apply_wounds(key)
+  end
+  pause 180 # pause 3 minutes before using scar remedies
```

Comment:
Why does this need to pause?

Reply:
The pause is so scars actually show from the use of the remedies so the scar remedies work.

Reply:
Do you need to `check_health` again or were scars already captured?

Reply:
No, grabbing the areas where the initial wounds were captured.